test/e2e/node/pod_resize.go

```diff
@@ -1408,14 +1441,133 @@ func doPodResizeErrorTests() {
 	}
 }
 
-var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
-	featureGatePostAlpha = true
-	if fs, found := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()[features.InPlacePodVerticalScaling]; found {
-		if fs.PreRelease == featuregate.Alpha {
-			featureGatePostAlpha = false
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-testcase1", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		if nodeNum > 1 {
+			framework.Logf("There are more than 1 node in the testing cluster, so skip the pod-in-queue-resize-test!")
+			return
 		}
-	}
+
+		ginkgo.By("Checking the node allocatable CPU resources!")
+		node := nodes.Items[0]
+		nodeAllocatableResources := node.Status.Allocatable
+		nodeAllocatableCPU := nodeAllocatableResources.Cpu()
+		nodeAllocatableCPUValue := nodeAllocatableCPU.MilliValue()
+		nodeAllocatableCPUStr := nodeAllocatableCPU.String()
+		framework.Logf("Node allocatable CPU is %s", nodeAllocatableCPUStr)
+		testPodRequest := nodeAllocatableCPUValue / 2
```

Comment:
Will this test work if there are other pods on the node?
Say Node allocatable is 4000 millicpus, but there is coreDNS pod or some other CNI daemon pods taking up 1000m. We probably need to look at nodeAllocatableCPUValue - (what is currently used), and divide that up between two pods.